buildAll.proj

```diff
@@ -16,7 +16,17 @@
                      src\Microsoft.ServiceFabric.Actors.Wcf\Microsoft.ServiceFabric.Actors.Wcf.csproj;
                      src\Microsoft.ServiceFabric.Services\Microsoft.ServiceFabric.Services.csproj;
                      src\Microsoft.ServiceFabric.Services.Remoting\Microsoft.ServiceFabric.Services.Remoting.csproj;
-                     src\Microsoft.ServiceFabric.Services.Wcf\\Microsoft.ServiceFabric.Services.Wcf.csproj"/>
+                     src\Microsoft.ServiceFabric.Services.Wcf\\Microsoft.ServiceFabric.Services.Wcf.csproj;" />
+
+    <ProjectsToBuildIncludingTests Include="src\FabActUtil\FabActUtil.csproj;
+                     src\Microsoft.ServiceFabric.Actors\Microsoft.ServiceFabric.Actors.csproj;
```

Comment:
Can these projects list be replaced with $(ProjectsToBuild); instead of copying it here again.